mev_inspect/classifiers/specs/aave.py

```diff
@@ -35,8 +78,36 @@ def get_transfer(trace: DecodedCallTrace) -> Transfer:
     protocol=Protocol.aave,
     classifiers={
         "transferOnLiquidation(address,address,uint256)": AaveTransferClassifier,
-        "transferFrom(address,address,uint256)": AaveTransferClassifier,
     },
 )
 
-AAVE_CLASSIFIER_SPECS = [AAVE_SPEC, ATOKENS_SPEC]
+AAVE_CLASSIFIER_SPECS: List[ClassifierSpec] = [AAVE_SPEC, ATOKENS_SPEC]
+
+
+def _get_received_data(
+    liquidation_trace: DecodedCallTrace,
+    child_transfers: List[Transfer],
+    liquidator: str,
+) -> Tuple[str, int]:
+
+    """Look for and return liquidator payback from liquidation"""
+    for transfer in child_transfers:
+
+        if transfer.to_address == liquidator:
+            return transfer.token_address, transfer.amount
+
+    return liquidation_trace.inputs["_collateral"], 0
+
+
+def _get_debt_data(
```

Comment:
same argument as for compound switching to `Optional[Transfer]`

Reply:
good call - will update to match compound